src/ngraph/ngraph_nnvm_utils.h

```diff
@@ -49,7 +49,9 @@ inline std::shared_ptr<TensorView> NDArray_to_TensorView(
   auto shape = TShape_to_NShape(input.shape());
   const auto& element_type = getType(input.dtype());
 
-  auto TV = backend->make_primary_tensor_view(element_type, shape);
+  // TODO(mbrookhart): I don't think Ashok's memory sharing PR has a
+  // create_tensor implementation? this will probably conflict
```

Comment:
currently , both API are supported, so there should be no conflict. I have a sprint task to update ngraph tensorview api for this change, and also look into layout issue.

Reply:
Thanks!